sdk/migrationdiscoverysap/Azure.ResourceManager.MigrationDiscoverySap/tests/Tests/Enums/ImportOperationState.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Azure.ResourceManager.MigrationDiscoverySap.Tests.Tests.Enums
+{
+    /// <summary>
+    /// ARM acceptable operation states.
+    /// <see href="https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#1325----operation-resource"/>.
+    /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum ImportOperationState
```

Comment:
let's ask arthur if there's a way to add this as a enum so that clients too can use.